locale/en_US/submission.xml

```diff
@@ -607,6 +607,8 @@
 	<message key="submission.query.lastReply">Last Reply</message>
 	<message key="submission.query.closed">Closed</message>
 	<message key="submission.query.addNote">Add Message</message>
+	<message key="submission.query.leaveQuery">Leave Discussion</message>
+	<message key="submission.query.leaveQuery.confirm">If you leave this conversation you will no longer be notified of new messages. Would you like to leave the conversation?</message>
```

Comment:
Yes, you're right. Use "discussion" instead of "conversation".

Reply:
check